src/main/scala/com/stratio/hermes/helpers/RandomHelper.scala

```diff
@@ -24,10 +24,27 @@ import scala.util.Random
 object RandomHelper {
 
   /**
-   * Returns a random element from a list.
-   * @param l initial list
-   * @tparam T with the type of the list
-   * @return a random element of the list or None if the list is empty.
-   */
+    * Returns a random element from a list.
+    *
+    * @param l initial list
+    * @tparam T with the type of the list
+    * @return a random element of the list or None if the list is empty.
+    */
   def randomElementFromAList[T](l: List[T]): Option[T] = Random.shuffle(l).headOption
+
+  /**
+    * Returns length of a Integer element.
+    *
+    * @param n number to calculate length.
+    * @return size of the integer.
+    */
+  def numberOfDigitsFromANumber(n: Int): Int = if (n == 0) 1 else math.log10(math.abs(n)).toInt + 1
```

Comment:
This method is only used for testing purposes?? If so I'd not include it in this class